Reference/V9-Config/ContentSettings/index.md

```diff
@@ -195,3 +200,39 @@ If you need to create a custom media document type to handle images called somet
   ]
 }
 ```
+
+## ContentVersionCleanupPolicy
+
+The global settings for the scheduled job which cleans up historic content versions, these settings can be overridden per document type.
+
+Current draft and published versions will never be removed, nor will individual content versions which have been marked as "preventCleanup".
+
+See [Content Version Cleanup](/documentation/Fundamentals/Data/Content-Version-Cleanup/index.md) for more details on overriding configuration and preventing cleanup of specific versions.
+
+```json
+"ContentVersionCleanupPolicy": {
+  "EnableCleanup": false,
+  "KeepAllVersionsNewerThanDays": 7,
+  "KeepLatestVersionPerDayForDays": 90
+}
+```
+
+If you don't wish to store any historic content versions at all simply set both of the "keep" settings values to 0.
```

Comment:
This is not completely correct. This will just delete them hourly.

Reply:
:cowboy_hat_face: